x-pack/plugins/security_solution/server/lists_integration/endpoint/validators/trusted_app_validator.ts

```diff
@@ -197,32 +196,26 @@ export class TrustedAppValidator extends BaseValidator {
 
   async validatePreDeleteItem(): Promise<void> {
     await this.validateHasWritePrivilege();
-    await this.validateCanManageEndpointArtifacts();
   }
 
   async validatePreGetOneItem(): Promise<void> {
     await this.validateHasReadPrivilege();
-    await this.validateCanManageEndpointArtifacts();
   }
 
   async validatePreMultiListFind(): Promise<void> {
     await this.validateHasReadPrivilege();
-    await this.validateCanManageEndpointArtifacts();
   }
 
   async validatePreExport(): Promise<void> {
-    await this.validateHasReadPrivilege();
-    await this.validateCanManageEndpointArtifacts();
+    await this.validateHasWritePrivilege();
```

Comment:
Is it intentional to check `write` privilege instead of the previous `read` privilege?

Reply:
Yes, it was a request from @caitlinbetz. Forgot to mention that on my previous comment. Thanks for pointing this out!

Reply:
Cool, thanks! 🚀